tests/model/test_model.py

```diff
@@ -744,6 +745,40 @@ def test__group_info_from_realm_user_groups(self, model,
             }
         assert user_group_names == ['Group 1', 'Group 2', 'Group 3', 'Group 4']
 
+    @pytest.mark.parametrize('presence, time_in_sec, expected_status', [
+        ('active',  0,       'Active now'),
+        ('active',  110,     'Active now'),
+        ('idle',    110,     'Just now'),
+        ('idle',    190,     '3 minutes ago'),
+        ('idle',    3600,    'An hour ago'),
+        ('idle',    10000,   '2 hours ago'),
+        ('idle',    100000,  'Yesterday'),
+        ('idle',    200000,  '2 days ago'),
```

Comment:
Something here isn't working, as I'm getting `-1 days ago` for idle users in the user list.

Reply:
Strange :thinking:. It's working fine for me locally.
There may be some timezone data mishandling in the Model method (on first look),
Or, there could be some unlooked loopholes in the below logic (in `helper`, last commit):
```python
    days = (datetime.now() - datetime.fromtimestamp(
            timestamp)).days
```
Thanks for reporting, as this could be highly misleading :+1:

Reply:
The 'ago' part can be left until later I suspect - ie. we could leave it as absolute date/time for now?

Reply:
Yes